packages/docs/src/examples/Transaction.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import { parseEther } from '@ethersproject/units'
+import { Mainnet, DAppProvider, useSendTransaction, useEthers, Config } from '@usedapp/core'
+import { ChainId } from '../../../core/src/constants/chainId'
```

Comment:
Watch it, a bad import. It won't work for the users.

Reply:
What is the funniest, it worked 😂